server/db/models/game.js

```diff
@@ -38,6 +38,7 @@ var gameSchema = new mongoose.Schema({
   deck: { type: [Number],
     default: [10,20,30,40,50,60,70,90,110,130,150,170,190,210,230,250,270,290,310,330,350,370,390,410,80,100,120,140,160,180,200,220,240,260,280,300,320,340,360,380,400,420,430,440,450,460,470,480,490,500,510,520,530,540,550,560,570,580,590,600,610,620,630,640,650,660,670,680,690,700,710,720,730,740,750,760,770,780,790,800,810,820,830,840]},
   discard: [Number],
+  numFlags: Number
```

Comment:
numFlags:  {
   type: Number,
   enum: [1, 2, 3, 4] //since there can be 4 flags max
}